meteor/rsmap.py

```diff
@@ -308,6 +309,23 @@ def from_gemmi(
             uncertainty_column=uncertainty_column,
         )
 
+    @classmethod
+    def from_numpy_map(
+        cls, map_grid: np.ndarray, *, spacegroup: Any, cell: Any, high_resolution_limit: float
+    ) -> Map:
```

Comment:
re: your comment in PR

I think even just `from_3Dnumpy_to_rsmap` would go a long way..